### PR TITLE
Bugfix/#1270 Fixing Number of project listed on org index includes active AND archived

### PR DESCRIPTION
**Pull request by pamungkaski**
#1270 
### Proposed changes in this pull request

- add test on organization/tests/test_views_default_organizations.py
- fix the bug on organization/views/default.py by filtering project__archived

### When should this PR be merged

- Anytime

### Risks

- Null

### Follow-up actions

- Null

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description. 
- [ ] **Is the PR labeled correctly?** It should have the `migration` label if a new migration is added. 
- [ ] **Is the risk level assessment sufficient?** The risks section should contain all risks that might be introduced with the PR and which actions we need to take to mitigate these risks. Possible risks are database migrations, new libraries that need to be installed or changes to deployment scripts.

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. 

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. 
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). 

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components. 
- [ ] **If this is a bug fix, are tests for the issue in place?**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature?** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?** 
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes must be documented in the [Cadasta Platform Documentation](https://github.com/Cadasta/cadasta-docs).
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented in the [API docs](https://github.com/Cadasta/api-docs).
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

**Comment by pamungkaski** [comment]
7f6ca4b87ee633ceb3663fb937810ed7d0816032
that is my actual bugfix code

File: cadasta/organization/tests/test_views_default_organizations.py [hunk 1]
```diff
@@ -63,6 +64,20 @@ def setup_template_context(self):
...
             'is_superuser': False,
         }
+    # Unit test to test the org.num_projects count
+    # only the active project
+
+    def test_should_only_active_projects_added_at_org_index(self):
+        org_list = self.view_class.queryset
+        num_init_proj = []
+        for i in range(org_list.count()):
+            num_init_proj.append(org_list[i].num_projects)
```
Comment (seav) [inline cadasta/organization/tests/test_views_default_organizations.py]: I'm not entirely convinced that this test is robust enough. If I remove the 2 lines with `ProjectFactory.create()`, this test will still pass. So we are not really sure that the existence of archived projects really does not affect the `num_projects` aggregate annotation.

Comment (pamungkaski) [inline cadasta/organization/tests/test_views_default_organizations.py]: Hi @seav thanks for your response
Which line did you mean? So i can check it and work on it immedietly

**Comment by seav** [comment]
Hi @pamungkaski, thank you for your work towards fixing this issue. We have accepted a PR from another developer to close the issue. I will close this PR now.

**Pull request closed without merging**